starter/app/src/main/java/com/example/android/politicalpreparedness/representative/RepresentativeViewModel.kt

```diff
@@ -1,12 +1,27 @@
 package com.example.android.politicalpreparedness.representative
 
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.android.politicalpreparedness.network.CivicsApiService
+import com.example.android.politicalpreparedness.network.models.Address
+import com.example.android.politicalpreparedness.representative.model.Representative
+import kotlinx.coroutines.launch
 
-class RepresentativeViewModel: ViewModel() {
+class RepresentativeViewModel(private val civicsApiService: CivicsApiService) : ViewModel() {
```

Comment:
Please don't use the service on ViewModel. Access the service via the Repository